src/main/scala/com/databricks/spark/csv/util/TypeCast.scala

```diff
@@ -35,7 +38,8 @@ object TypeCast {
    * @param datum string value
    * @param castType SparkSQL type
    */
-  private[csv] def castTo(datum: String, castType: DataType, nullable: Boolean = true): Any = {
+  private[csv] def castTo(datum: String, castType: DataType, nullable: Boolean = true,
+                          locale: Locale = Locale.getDefault): Any = {
```

Comment:
This does not seem to be passed anywhere. That is who is going to pass ```local``` to castTo?